src/spdx_matcher/__init__.py

```diff
@@ -16,6 +16,18 @@
    limitations under the License.
 """
 
+from __future__ import annotations
```

Comment:
`__future__` should be the first line of source code, So I put it in L19, also I move others import syntax near to L19